pennylane/ftqc/__init__.py

```diff
@@ -26,7 +26,7 @@
 
 """
 from .operations import RotXZX
-from .decomposition import convert_to_mbqc_gateset
+from .decomposition import convert_to_mbqc_formalism, convert_to_mbqc_gateset
```

Comment:
Nothing to be done for now, but I'm wondering if we drop the `formalism` in the long term.